src/test/java/com/solvd/LoginTest.java

```diff
@@ -0,0 +1,22 @@
+package com.solvd;
+
+import com.solvd.pages.AccountPage;
+import com.solvd.pages.HomePage;
+import com.solvd.pages.LoginPage;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class LoginTest extends AbstractTest{
+//    CID4
+    @Test
+    public void testCorrectLogin() {
+        HomePage homePage = new HomePage(getDriver());
+        homePage.navigateToHomePage();
+
+        LoginPage loginPage = homePage.clickLogin();
+        loginPage.enterLogin("autologin");
+        loginPage.enterPassword("autopassword");
+        AccountPage accountPage = loginPage.clickAccount();
```

Comment:
Create in Login page performLogin() method which avoids duplicate login steps in future tests